service_test.go

```diff
@@ -0,0 +1,114 @@
+package certificatetpr
+
+import (
+	"testing"
+	"time"
+
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/giantswarm/micrologger/microloggertest"
+)
+
+// Test_NewService tests the NewService function.
+func Test_NewService(t *testing.T) {
```

Comment:
This is largely just some junk testing. Made me feel good about myself, and does help with some validation.